testing/cli/test_cli_sct_analyze_lesion.py

```diff
@@ -51,6 +51,12 @@ def compute_expected_measurements(dim, starting_coord=None, path_seg=None):
         min_area = min(np.sum(data_seg[:, n_slice, :])
                        for n_slice in range(starting_coord[1], starting_coord[1] + dim[1]))
 
+        # Find the midsagittal slice of the spinal cord
+        # Note: we use [2] here although in sct_analyze_lesion we use [0]
+        #  (because sct_analyze_lesion does RPI reorientation)
+        mid_sagittal_slice = int(np.mean([np.min(np.unique(np.where(data_seg)[2])),
+                                          np.max(np.unique(np.where(data_seg)[2]))]))
+
```

Comment:
Just some small simplifying changes (same as in `sct_analyze_lesion`):

```python
# Find the midsagittal slice of the spinal cord (assuming AIL input image)
nonzero_slices = np.unique(np.where(data_seg)[2])  # AIL image: [2] -> LR (sagittal)
mid_sagittal_slice = int(np.mean([np.min(nonzero_slices), np.max(nonzero_slices)]))
```

Reply:
Thanks, done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4617/commits/d8af57d8d0db932a5c3496f7a38b7cce0ac3ccf6